app/src/main/java/com/ciandt/dragonfly/example/helpers/NotificationHelper.kt

```diff
@@ -0,0 +1,65 @@
+package com.ciandt.dragonfly.example.helpers
+
+import android.app.Notification
+import android.app.PendingIntent
+import android.content.Context
+import android.support.v4.app.NotificationCompat
+import android.support.v4.app.NotificationManagerCompat
+import android.support.v4.content.ContextCompat
+import com.ciandt.dragonfly.example.R
+import com.ciandt.dragonfly.example.features.download.DownloadedFile
+
+object NotificationHelper {
```

Comment:
Since all these methods seem to be related to download, I think it would be better to rename the class to DownloadNotificationHelper